src/App.jsx

```diff
@@ -5,17 +5,25 @@ import { useState } from "react"
 import { createPortal } from "react-dom"
 import { RulesModal } from "./components/RulesModal/RulesModal"
 import { NavBar } from "./components/NavBar/NavBar"
+import { ResultsModal } from "./components/ResultsModal/ResultsModal"
 
 function App() {
     const weldorInstance = useWeldor()
     const [showRules, setShowRules] = useState(true)
+    const [showResults, setShowResults] = useState(true)
 
     return (
       <>
         {showRules && createPortal(
```

Comment:
Please make sure this can't be triggered when the results are shown.
![image](https://github.com/user-attachments/assets/5a7214c5-80f7-42f6-91af-20d23f442ec6)

Reply:
Wait GH screwed up the reference, I meant line 17.

Reply:
Changes applied, thanks.